FishjamClient/src/main/java/com/fishjam/client/FishjamClient.kt

```diff
@@ -12,15 +12,15 @@ import org.membraneframework.rtc.models.RTCStats
 import org.membraneframework.rtc.utils.Metadata
 import org.webrtc.Logging
 
-typealias TrackContext = JellyfishTrackContext
+typealias TrackContext = FishjamTrackContext
```

Comment:
_(as part of other PR - I would check if FishjamTrackContext can be removed, as it happened on iOS version)_

Reply:
I had that in mind. I [ticket created](https://membraneframework.atlassian.net/jira/software/projects/EX/boards/24?selectedIssue=EX-528).